test/features/visiting_the_article_index_test.rb

```diff
@@ -0,0 +1,21 @@
+require "test_helper"
+
+feature "Visiting The Article Index" do
+
+  scenario "lists all posts on the articles page" do
+    # Given that there are multiple posts
+    # When a user visits the page
+    # Then all of the blog posts should be displayed
+  end
+
+end
+
+feature "Visiting the Post Index" do
+  scenario 'With an existing post' do
+    # Given that a post exists
+    # When a user visits the articles page
+    visit articles_path
+    # Then a new page should load displaying that post
+    page.text.must_include 'Hello World'
+  end
+end
```

Comment:
For the index page, you'd want to verify that multiple articles are showing.  You might want to create two unique Articles (explicitly, instead of using Faker) and then assert their existence on the page.  The purpose of the index page is to show you all of a resource - that's not being tested by asserting the existence of "Hello World" text.

```ruby
Article.create!(title: "...", content: "...")
Article.create!(title: "...", content: "...")
```